Remotion/Web/Development.WebTesting/Configuration/TestInfrastructureConfiguration.cs

```diff
@@ -39,15 +39,15 @@ public class TestInfrastructureConfiguration : ITestInfrastructureConfiguration
     private readonly IRequestErrorDetectionStrategy _requestErrorDetectionStrategy;
     private readonly bool _closeBrowserWindowsOnSetUpAndTearDown;
 
-    public TestInfrastructureConfiguration ([NotNull] WebTestConfigurationSection webTestConfigurationSection)
+    public TestInfrastructureConfiguration ([NotNull] IWebTestConfiguration webTestConfigurationSection)
     {
       ArgumentUtility.CheckNotNull("webTestConfigurationSection", webTestConfigurationSection);
 
       _webApplicationRoot = webTestConfigurationSection.WebApplicationRoot;
       _screenshotDirectory = webTestConfigurationSection.ScreenshotDirectory;
 
       _closeBrowserWindowsOnSetUpAndTearDown = webTestConfigurationSection.CloseBrowserWindowsOnSetUpAndTearDown;
-      _requestErrorDetectionStrategy = GetRequestErrorDetectionConfiguration(webTestConfigurationSection.RequestErrorDetectionStrategyTypeName);
```

Comment:
Maybe we should keep the original `TypeName` suffix for this property. This makes it clearer that it contains just the type name (string) instead of an actual strategy object. What do you think?

Reply:
Yes, I think its better with the suffix